R/verbatim_popup.R

```diff
@@ -37,13 +37,9 @@ verbatim_popup_ui <- function(id, button_label, type = c("button", "link"), ...)
   ns <- shiny::NS(id)
   ui_args <- list(
     inputId = ns("button"),
-    label = div(
-      button_label,
-      shiny::uiOutput(ns("disable_controller"), inline = TRUE) # this is dummy output - see the comment in srv
-    )
+    label = button_label
```

Comment:
@gogonzo the warning message is impossible here now. The button and label are static.
Please test it once more.

Reply:
It's working, there was something wrong on my side.